tests/regression/runner.py

```diff
@@ -562,6 +563,48 @@ def test_find_transactions_obj_by_tag(self):
 
         eval_stat(time_cost, "find transactions obj by tag")
 
+    def test_get_tips(self):
+        logging.debug(
+            "\n================================get_tips================================"
+        )
+        # cmd
+        #    0. call get_tips normally
+        #    1. call get_tips with unwanted ascii string
+        #    2. call get_tips with unwanted unicode string
+        rand_tag_27 = gen_rand_trytes(27)
+        test_cases = ["", rand_tag_27, "飛天義大利麵神教"]
```

Comment:
I wonder if `test_cases` is a meaningful name.

Reply:
Is `query_data` a mote suitable name?

Reply:
I think so. Alternatively, `query_string` works as well.